src/Platform.Handlers/src/Xamarin.Platform.Handlers/PropertyMapper.cs

```diff
@@ -0,0 +1,109 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Xamarin.Platform
+{
+	public class PropertyMapper {
```

Comment:
is there a use case for this to be subclassed by users (with part of the API being internal) ?

Reply:
We're subclassing it for the Generic mapper here

https://github.com/xamarin/Xamarin.Forms/blob/83a7ad4f3e5bbdc4a4fbc97965617710e6c9052d/src/Platform.Handlers/src/Xamarin.Platform.Handlers/PropertyMapper.cs#L34

So yes